packages/web/src/components/ui/PackagePreview/PackagePreview.tsx

```diff
@@ -59,10 +59,11 @@ export default function PackagePreview({
               </span>
             )}
           </div>
-
+          {/*
```

Comment:
Can we keep just this as a comment with a TODO and delete `open` altogether for now? This seems quite heavy to read and wouldn't be hard to restore after.